src/main/kotlin/no/nav/familie/ef/sak/behandlingsflyt/steg/BeregnYtelseSteg.kt

```diff
@@ -173,7 +173,7 @@ class BeregnYtelseSteg(
     ) {
         validerAntallBarnOgUtgifterVedMidlertidigOpphør(utgiftsperioder, saksbehandling.id)
         validerTidligereVedtakVedMidlertidigOpphør(utgiftsperioder, saksbehandling)
-        validerSammenhengendePerioderVedMidlertidigOpphør(utgiftsperioder, saksbehandling)
+        validerSammenhengendePerioderVedMidlertidigOpphør(utgiftsperioder)
```

Comment:
Ser fint ut, kanskje kan endre navn på denne nå då den ikke har noe å gjøre med midlertidigOpphør noe lengre? :) 
```suggestion
        validerSammenhengendePerioder(utgiftsperioder)
```

Reply:
(husk å endre selve metoden og!)